itou/utils/templatetags/buttons_form.py

```diff
@@ -79,6 +79,18 @@ def itou_buttons_form(
     if any(matomo_values) and not all(matomo_values):
         raise ValueError("Matomo values are all or nothing")
 
+    if (primary_name is None) != (primary_value is None):
```

Comment:
Un pattern que j'aime bien pour tester ce genre de truc c'est : 
```suggestion
    if {primary_name is None, primary_value is None} == {True, False}:
```